lib/workers/repository/update/branch/index.ts

```diff
@@ -520,15 +521,21 @@ export async function processBranch(
         await embedChangelogs(config.upgrades);
       }
 
+      config.updatedArtifacts ??= [];
+      config.artifactErrors ??= [];
+
       const postUpgradeCommandResults =
         await executePostUpgradeCommands(config);
 
       if (postUpgradeCommandResults !== null) {
         const { updatedArtifacts, artifactErrors } = postUpgradeCommandResults;
-        config.updatedArtifacts = updatedArtifacts;
-        config.artifactErrors = artifactErrors;
+        config.updatedArtifacts.push(...updatedArtifacts);
+        config.artifactErrors.push(...artifactErrors);
```

Comment:
can't this cause duplicates?

Reply:
This is an accidental change brought in from a refactor I have looked into it. 

Though I have landed on that we probably should move `updatedArtifacts` and `updatedPackageFiles` to maps with the path as key and value `FileChange` 

Reverting